mlrun/utils/helpers.py

```diff
@@ -1266,6 +1266,36 @@ def is_legacy_artifact(artifact):
         return not hasattr(artifact, "metadata")
 
 
+def get_format_run(run: list, with_project=False):
+    fields = [
+        "id",
+        "name",
+        "status",
+        "error",
+        "created_at",
+        "scheduled_at",
+        "finished_at",
+        "description",
+        "project" if with_project else "",
```

Comment:
that will leave an empty key. Id suggest extending the fields by conditioning if `with_project`

Reply:
it wouldn't, cause later if the field isn't in the object it will not include it, but i'll change it to be safe